examples/plot_decoding_tutorial.py

```diff
@@ -130,22 +107,24 @@
 #
 # As a decoder, we use a Support Vector Classification, with a linear
 # kernel.
+# We will use The mask is a mask of the Ventral Temporal streaming coming from
+# the Haxby study.
 #
 # We first create it:
-from sklearn.svm import SVC
-svc = SVC(kernel='linear')
-print(svc)
+from nilearn.decoding import Decoder
+mask_filename = haxby_dataset.mask_vt[0]
```

Comment:
mask is already defined above. May be you want to remove this one or above one ?